cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,85 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const faker = require('faker');
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
+
 describe('Settings page', () => {
-  before(() => {
+  let user;
+
+  const userChanged = {
+    username: faker.name.firstName(),
+    bio: faker.lorem.words(),
+    email: faker.internet.email(),
+    password: faker.internet.password(),
+  };
 
+  const successUpdate = 'Update successful!';
+
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    homePage.visit();
+    cy.login(user.email, user.username, user.password);
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage.visit();
+    settingsPage.updateUsernameField(userChanged.username);
+    settingsPage.clickUpdateSetBtn();
+    settingsPage.assertSuccessfulUpdate(successUpdate);
+    settingsPage.clickOnModalOkBtn();
 
+    homePage.assertUsername(userChanged.username);
```

Comment:
Each test should be independent of the others. Here, the test for updating the username relies on the home page to check the result of the operation. Consider making the test self-contained by checking the result on the settings page itself.